app/templates/home.hbs

```diff
@@ -67,20 +67,26 @@
 
           <h2>Decklist of the week</h2>
 
-          <DecklistBox
-            @decklist={{@model.dotw}}
-            @showTitle="true"
-            @showLink="true"
-            @showAuthor="true"
-          />
+          {{#if @mode.dotw}}
```

Comment:
@model.dotw

Reply:
Oh jaysus! Thank heavens for code reviews. Fixed!